openquake/hazardlib/site.py

```diff
@@ -45,9 +45,6 @@ class Site(object):
     :param z2pt5:
         Vertical distance from earth surface to the layer where seismic waves
         start to propagate with a speed above 2.5 km/sec, in km.
-    :param backarc":
-        Boolean value, ``True`` if the site is in the subduction backarc and
-        ``False`` if it is in the subduction forearc or is unknown
```

Comment:
I can see it makes sense to remove the "optional" parameters from the doc strings here, but is it possible to put somewhere into the API documentation the list of recognised site parameters and their units?

Reply:
See https://docs.openquake.org/oq-engine/master/openquake.hazardlib.html#openquake.hazardlib.site.SiteCollection